src/fmu/dataio/_models/_schema_base.py

```diff
@@ -138,9 +138,9 @@ def url(cls) -> str:
         DEV_URL = f"{FmuSchemas.DEV_URL}/{cls.PATH}"
         PROD_URL = f"{FmuSchemas.PROD_URL}/{cls.PATH}"
 
-        if os.environ.get("SCHEMA_RELEASE", False):
-            return PROD_URL
-        return DEV_URL
+        if os.environ.get("DEV_SCHEMA", False):
+            return DEV_URL
+        return PROD_URL
```

Comment:
We won't be able to produce metadata with the `$schema = radix dev` without exploring some other option. I guess this is OK but not ideal